IdentityCore/src/util/NSURL+MSIDExtensions.m

```diff
@@ -141,7 +141,7 @@ - (NSURL *)msidURLForHost:(NSString *)preferredHost context:(id<MSIDRequestConte
             {
                 // setPercentEncodedHost and setPort both throw if there's an error, so it's okay for
                 // us to throw here as well to propogate the error
-                @throw [NSException exceptionWithName:@"InvalidNumberFormatException" reason:@"Port is not a valid integer or port" userInfo:nil];
+                @throw MSIDException(MSIDGenericException, @"Port is not a valid integer or port.", nil);
```

Comment:
This looks like a potentially breaking change. Possible to use previous exception info?